### PR TITLE
Add quantization API doc and oneDNN to migration guide

### PR DESCRIPTION
**Pull request by bgawrych**
## Description ##
Migration guide:
image

Quantization API:
image

### CODE REVIEW

**Comment by bgawrych** [comment]
@mxnet-bot run ci [windows-gpu]

**Comment by bgawrych** [comment]
@mxnet-bot run ci [windows-gpu]

**Comment by bgawrych** [comment]
@mxnet-bot run ci [windows-gpu]

File: docs/python_docs/python/tutorials/getting-started/gluon_migration_guide.md [hunk 1]
```diff
@@ -432,6 +432,67 @@ A new module called `mxnet.gluon.probability` has been introduced in Gluon 2.0.
 
 3. [Transformation](https://github.com/apache/incubator-mxnet/tree/master/python/mxnet/gluon/probability/transformation): implement invertible transformation with computable log det jacobians.
 
+##  oneDNN Integration
+### Operator Fusion
+In versions 1.x of MXNet pattern fusion in execution graph was enabled by default when using MXNet built with oneDNN library support and could have been disabled by setting 'MXNET_SUBGRAPH_BACKEND' environment flag to `None`. MXNet 2.0 introduced changes in forward inference flow which led to refactor of fusion mechanism. To fuse model in MXNet 2.0 there are two requirements:
+
+ - the model must be defined as a subclass of HybridBlock or Symbol
```
Comment (bartekkuncer) [inline docs/python_docs/python/tutorials/getting-started/gluon_migration_guide.md:439]:
```suggestion
 - the model must be defined as a subclass of HybridBlock or Symbol,
```

File: docs/python_docs/python/tutorials/getting-started/gluon_migration_guide.md [hunk 2]
```diff
@@ -432,6 +432,67 @@ A new module called `mxnet.gluon.probability` has been introduced in Gluon 2.0.
...
+
+ - the model must have specific operator patterns which can be fused
```
Comment (bartekkuncer) [inline docs/python_docs/python/tutorials/getting-started/gluon_migration_guide.md:441]:
```suggestion
 - the model must have specific operator patterns which can be fused.
```

File: docs/python_docs/python/tutorials/getting-started/gluon_migration_guide.md [hunk 3]
```diff
@@ -432,6 +432,67 @@ A new module called `mxnet.gluon.probability` has been introduced in Gluon 2.0.
...
+
+Both HybridBlock and Symbol classes provide API to easily run fusion of operators. All we have to do is to add single line of code running fusion passes on our model:
```
Comment (bartekkuncer) [inline docs/python_docs/python/tutorials/getting-started/gluon_migration_guide.md:443]: Only sentence written in first person.

File: python/mxnet/contrib/quantization.py [hunk 4]
```diff
@@ -174,15 +174,16 @@ def __init__(self):
...
-            arr : NDArray
-                NDArray containing data of monitored node
+
+        Parameters
+        ----------
+        name : str
+            Node name from which collected data comes from
+        op_name : str
+            Operator name from which collected data comes from. Single operator
+            can have multiple inputs/ouputs nodes - each should have different name
```
Comment (bartekkuncer) [inline python/mxnet/contrib/quantization.py:184]:
```suggestion
            can have multiple input/ouput nodes - each should have different name.
```

File: python/mxnet/contrib/quantization.py [hunk 5]
```diff
@@ -174,15 +174,16 @@ def __init__(self):
...
-            op_name : str
-                Operator name from which collected data comes from. Single operator
-                can have multiple inputs/ouputs nodes - each should have different name
-            arr : NDArray
-                NDArray containing data of monitored node
+
+        Parameters
+        ----------
+        name : str
+            Node name from which collected data comes from
```
Comment (bartekkuncer) [inline python/mxnet/contrib/quantization.py:181]:
```suggestion
            Node name from which collected data comes from.
```

File: python/mxnet/contrib/quantization.py [hunk 6]
```diff
@@ -174,15 +174,16 @@ def __init__(self):
...
+        op_name : str
+            Operator name from which collected data comes from. Single operator
+            can have multiple inputs/ouputs nodes - each should have different name
+        arr : NDArray
+            NDArray containing data of monitored node
```
Comment (bartekkuncer) [inline python/mxnet/contrib/quantization.py:186]:
```suggestion
            NDArray containing data of monitored node.
```

**Review by bartekkuncer — CHANGES_REQUESTED** [review]
You can also add dot at the end of the sentence in python/mxnet/contrib/quantization.py:166.

File: docs/python_docs/python/tutorials/getting-started/gluon_migration_guide.md (hunk 3 above)
Comment (bgawrych) [inline docs/python_docs/python/tutorials/getting-started/gluon_migration_guide.md:443]:
```suggestion
Both HybridBlock and Symbol classes provide API to easily run fusion of operators. Adding only single line of code is needed to run fusion passes on model:
```

**Comment by bgawrych** [comment]
@mxnet-bot run ci [windows-gpu, unix-cpu]

**Comment by bgawrych** [comment]
@mxnet-bot run ci [unix-cpu]

**Comment by bgawrych** [comment]
@mxnet-bot run ci [unix-cpu]

**Review by bartekkuncer — APPROVED** [review]
LGTM. Well written.

**Review by agrabows — APPROVED** [review]

**Pull request merged**